src/conv-core/isomalloc.C

```diff
@@ -168,6 +182,8 @@ static int init_map() { return 0; /*Isomalloc never works without mmap*/ }
 CpvStaticDeclare(int, zerofd); /*File descriptor for /dev/zero, for mmap*/
 #endif
 
+static constexpr void * const mmap_fail = (void *)~(uintptr_t)0;
```

Comment:
the value of mmap_fail is different here from the value for _Win32. Is this intentional?

Reply:
Yes, it varies by implementation. mmap returns -1, VirtualAlloc returns null.

Reply:
Ok, can you add a brief comment there to explain this? Also, could this be `MAP_FAILED` instead of -1?

Reply:
Yes to both, done